.prettierrc

```diff
@@ -1,5 +1,6 @@
 {
   "singleQuote": true,
   "trailingComma": "all",
-  "printWidth": 100
+  "printWidth": 100,
+  "proseWrap": "always"
```

Comment:
Looks like this will affect our YAML normalization as well. The impact might be okay on its own, but affects a large number of files that I don't care to deal with here, so I'm just going to revert the configuration.